lib/virtio_mmio/virtio_serial_drv.c

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2022 Wind River Systems, Inc.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * $FreeBSD$
+ */
+
+#include <openamp/open_amp.h>
+#include <openamp/virtqueue.h>
+#include <openamp/virtio.h>
+#include <openamp/virtio_mmio.h>
```

Comment:
virtio serial driver shouldn't call the transport specific api(mmio) directly.

Reply:
Virtio drivers no longer include mmio-specific header files.